conf/cassandra.yaml

```diff
@@ -1688,6 +1683,11 @@ drop_compact_storage_enabled: false
 # Valid values are in [1, max available disk size of all data directories].
 # Defaults to null to disable and use the physically available disk size of data directories during calculations.
 # data_disk_usage_max_disk_size:
+# Guardrail to warn or fail when the minimum replication factor is lesser than threshold.
+# This would also apply to system keyspaces.
+# Suggested value for use in production: 2 or higher
+# minimum_rf_warn_threshold: -1
+# minimum_rf_fail_threshold: -1
```

Comment:
Didn't we say that we preferred `minimum_replication_factor_fail_threshold`? That would also apply to the getters/setters around, that should be:
* `getMinimumReplicationFactorWarnThreshold`
* `getMinimumReplicationFactorFailThreshold`
* `getMinimumReplicationFactor`